components/operator/internal/state/installed_status.go

```diff
@@ -72,9 +74,9 @@ func updateStatus(ctx context.Context, r *reconciler, s *systemState) error {
 		{pulladdress, &s.instance.Status.InternalAccess.PullAddress, "Internal pull address", ""},
 		{pushAddress, &s.instance.Status.InternalAccess.PushAddress, "Internal push address", ""},
 		{registry.InternalAccessSecretName, &s.instance.Status.InternalAccess.SecretName, "Name of secret with registry access data", ""},
-		storageField,
 		pvcField,
 	}...)
+	fields = append(fields, storageFields...)
```

Comment:
Couldn't `storageFields...` be added in the previous append as well?
This is a tiny nitpick

Reply:
no, because there is one append with the `...` operator at the end of its arguments